baseplate/metrics.py

```diff
@@ -261,6 +272,24 @@ def decrement(self, delta=1, sample_rate=1.0):
         """
         self.increment(delta=-delta, sample_rate=sample_rate)
 
+class Histogram(object):
+    """Record of arbitrary values for distribution over time.
+
+    This implementation depends on histograms being created for
+    timer values in the supporting StatsD backend.
+    """
+    def __init__(self, transport, name):
+        self.transport = transport
+        self.name = name
+
+    def log(self, value):
+        """Log a new value to the histogram.
+
+        This records a new value to the histogram; the bucket it goes in
+        is determined by the backend service configurations.
+        """
+        serialized = self.name + (":{:g}|ms".format(value).encode())
```

Comment:
is it worth using the `h` alias just for explicitness?

Reply:
ah yeah, sure